cluster/test-deploy/Makefile

```diff
@@ -2,8 +2,12 @@ LATEST = $(shell curl -q "https://storage.googleapis.com/origin-ci-test/releases
 REPO ?= $(LATEST)
 PROFILE ?= "gcp-dev"
 TYPE ?= $(shell cat $(PROFILE)/.type)
+# Namespaces of images, azure requires to specify resource group
+TO_NS ?= ""
```

Comment:
Not suer I want this this way.  These should be in your vars unless there is a reason that doesn't work.